examples/misc/anchored_artists.py

```diff
@@ -11,7 +11,8 @@
 
 
 class AnchoredText(AnchoredOffsetbox):
```

Comment:
I wonder if we need this `class` when there's a `matplotlib.offsetbox.AnchoredText` implementation already?